Gemfile

```diff
@@ -1,17 +1,18 @@
 source 'https://rubygems.org'
 
 gem 'berkshelf', '~> 4.3'
-gem 'chef', '~> 12.4'
+gem 'chef', '~> 12.4', '<= 12.13.37'
```

Comment:
Is there another reason to lock this down? My reason was that newer versions deprecate compat_resource, so it spits a bunch of warnings, but that's not a very solid reason as it seems to still test okay.